common/persistence/sql/class.QueryIterator.php

```diff
@@ -14,99 +14,78 @@
  * along with this program; if not, write to the Free Software
  * Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
  *
- * Copyright (c) 2002-2008 (original work) 2014 Open Assessment Technologies SA
+ * Copyright (c) 2002-2020 (original work) 2014 Open Assessment Technologies SA
  *
  */
 
 /**
  * Iterator over all triples
  *
  * @author joel bout <joel@taotesting.com>
- * @package generis
  */
 class common_persistence_sql_QueryIterator implements Iterator
 {
-    const CACHE_SIZE = 100;
-    
+    private const CACHE_SIZE = 100;
+
     /**
      * @var common_persistence_SqlPersistence
      */
     private $persistence;
-    
-    /**
-     * Query to iterator over
-     *
-     * @var string
-     */
```

Comment:
Why did you remove them?

Reply:
@boajer  As we do not have php7.4 we still need `@var` php doc for attributes. When you put it back, I would remove the useless comments and fix the `$params` which is an array, not a string.